mimic/rest/maas_api.py

```diff
@@ -533,28 +533,46 @@ def create_alarm(self, request, tenant_id, entity_id):
         return ''
 
     @app.route('/v1.0/<string:tenant_id>/entities/<string:entity_id>/alarms/<string:alarm_id>',
+               methods=['GET'])
+    def get_alarm(self, request, tenant_id, entity_id, alarm_id):
+        """
+        Gets an alarm by ID.
```

Comment:
Ooh, and I missed a great opportunity to make an argument from authority - this is actually part of [PEP 257](https://www.python.org/dev/peps/pep-0257/), apparently:

> The docstring is a phrase ending in a period. It prescribes the function or method's effect as a command ("Do this", "Return that"), not as a description; e.g. don't write "Returns the pathname ...".